vcmiqt/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+set(vcmiqt_SRCS
+		StdInc.cpp
+
+		jsonutils.cpp
+		launcherdirs.cpp
+)
+
+set(vcmiqt_HEADERS
+		StdInc.h
+
+		jsonutils.h
+		launcherdirs.h
+		convpathqstring.h
+)
+
+assign_source_group(${vcmiqt_SRCS} ${vcmiqt_HEADERS})
+add_library(vcmiqt STATIC ${vcmiqt_SRCS} ${vcmiqt_HEADERS})
```

Comment:
it should be static only for "single app" builds. "Normal" desktop builds have launcher and editor as separate processes, so it's better to share code via dynamic (shared) library.